src/widgets/window.js

```diff
@@ -258,7 +259,7 @@ var FlatsealWindow = GObject.registerClass({
             showPermissionsButton = false;
         }
 
-        const isFolded = this._headerLeaflet.folded;
+        const isFolded = this._contentLeaflet.folded;
         if (isFolded) {
             showApplicationsButton = true;
             showPermissionsButton = true;
```

Comment:
Minor issue:

![image](https://user-images.githubusercontent.com/481418/105614475-6ed0d200-5da8-11eb-9789-b258af6d6f19.png)

Is that always-like-is-hovering effect intended on these buttons?

Reply:
Yes

Reply:
That's the .list-button style class